lib/list_item/twake_list_item.dart

```diff
@@ -4,21 +4,26 @@ import 'package:linagora_design_flutter/style/linagora_divider_style.dart';
 class TwakeListItem extends StatelessWidget {
   final Widget child;
   final double? height;
+  final EdgeInsetsGeometry? padding;
 
   const TwakeListItem({
     Key? key,
     required this.child,
     this.height,
+    this.padding,
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      height: height,
-      decoration: BoxDecoration(
-        border: LinagoraDividerStyle.material().borderDecoration,
+    return Padding(
+      padding: padding ?? EdgeInsets.zero,
```

Comment:
Why not use Container `padding` and `margin` properties?

Reply:
Updated